CRM/Contact/BAO/Query.php

```diff
@@ -5644,7 +5644,12 @@ public static function buildClause($field, $op, $value = NULL, $dataType = NULL)
         $value = CRM_Utils_Type::escape($value, $dataType);
         // if we don't have a dataType we should assume
         if ($dataType == 'String' || $dataType == 'Text') {
-          $value = "'" . strtolower($value) . "'";
```

Comment:
I think we just remove the strtolower & maybe add another unit test rather than adding more code as it has been shown to only hurt the db